sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -214,14 +214,14 @@ class RocksDB(
 
   /**
    * Remove RocksDB column family, if exists
+   * @return columnFamilyId if it exists, else None
```

Comment:
We do ensure that this `vcfId` is not used in the future right ?

Reply:
yeah, this case is dealt with in this [test case](https://src.dev.databricks.com/databricks/runtime@86ee602e873e6dcf0d9a4a9f1bd50302f6328598/-/blob/sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreSuite.scala?L1766)